### PR TITLE
Issue159

### PR DESCRIPTION
**Pull request by tproduit**
#159

### CODE REVIEW

**Comment by tproduit** [comment]
@sjib The images are renamed with the extension in lowercase.

**Review by sjib — APPROVED** [review]

**Comment by tproduit** [comment]
Add the documentation to solve issue #163
Check Sphinx syntax

File: en/qgep_swmm/Developpers-Advanced-Users.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+.. _Developpers-Advanced-Users:
+
+For developpers and advanced users
+==================================
+
+If you don't want to start QGIS each time that you want to run QGEP-SWMM functionnalities, you can also call them directly from a python script.
+A template script is provided in `src\swmm_templates\example_qgep_swmm_commands.py`
```
Comment (sjib) [inline en/qgep_swmm/Developpers-Advanced-Users.rst:7]: This path does not display correct in my preview: / are missing

File: en/qgep_swmm/Import-Results.rst [hunk 2]
```diff
@@ -3,11 +3,25 @@
...
+and they populate a view dedicated for the mapping (see :ref:`mapping-imported`).
+
+.. figure:: images/qgep_od_measurement_tables.jpg
+
 Launch ``SWMM Import results``:
 
 - ``SWMM report file``: the ``.rpt`` file generated during the previous step
 - ``Database``: The name of the PostgreSQL service hosting the QGEP database.
-- ``Simulation name``: The name that you want to give to this simulation.
+- ``Simulation name``: The name that you want to give to this simulation (it will appear in ``qgep_od.measuring_point.remark``).
```
Comment (sjib) [inline en/qgep_swmm/Import-Results.rst:23]: @tproduit And what defines the identifier of the connected measurement_series?

Comment (sjib) [reply en/qgep_swmm/Import-Results.rst:23]: @tproduit Maybe a sample screenshot of the measurement tables with data could be helpful to understand what is added where

**Pull request merged**